docs/get-started/java.md

```diff
@@ -1,131 +1,258 @@
+        );
+    }
+}
+```
+
+### Configure project and dependencies
 
-Add the following dependency to your `pom.xml` file:
+An ADK agent project requires this dependency in your
+`pom.xml` project file:
 
-```xml
+```xml title="my_agent/pom.xml (partial)"
 <dependencies>
     <dependency>
         <groupId>com.google.adk</groupId>
         <artifactId>adk-core</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
+        <version>0.3.0</version>
     </dependency>
 </dependencies>
 ```
 
-## Set up your API key
+Update the `pom.xml` project file to include this dependency and
+addtional settings with the following configuration code:
+
+??? info "Complete `pom.xml` configuration for project"
+    The following code shows a complete `pom.xml` configuration for
+    this project:
+
+    ```xml title="my_agent/pom.xml"
+    <?xml version="1.0" encoding="UTF-8"?>
+    <project xmlns="http://maven.apache.org/POM/4.0.0"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+        <modelVersion>4.0.0</modelVersion>
+
+        <groupId>com.example.agent</groupId>
+        <artifactId>adk-agents</artifactId>
+        <version>1.0-SNAPSHOT</version>
+
+        <!-- Specify the version of Java you'll be using -->
+        <properties>
+            <maven.compiler.source>17</maven.compiler.source>
+            <maven.compiler.target>17</maven.compiler.target>
+            <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        </properties>
+
+        <dependencies>
+            <!-- The ADK core dependency -->
+            <dependency>
+                <groupId>com.google.adk</groupId>
+                <artifactId>google-adk</artifactId>
+                <version>0.3.0</version>
+            </dependency>
+            <!-- The ADK dev web UI to debug your agent -->
+            <dependency>
+                <groupId>com.google.adk</groupId>
+                <artifactId>google-adk-dev</artifactId>
+                <version>0.3.0</version>
+            </dependency>
+        </dependencies>
+
+    </project>
+    ```
+
+### Set your API key
 
-This starter project uses the Gemini API, which requires an API key. If you
+This project uses the Gemini API, which requires an API key. If you
 don't already have Gemini API key, create a key in Google AI Studio on the 
 [API Keys](https://aistudio.google.com/app/apikey) page.
 
-In a terminal window, write your API key into an `.env` file as an environment variable:
+In a terminal window, write your API key into your `.env` file of your project
+to set environment variables:
 
 === "Windows"
 
-    ```shell
+    ```console title="Update: my_agent/.env"
     echo 'GOOGLE_API_KEY="YOUR_API_KEY"' > .env
     ```
 
 === "MacOS / Linux"
 
-    ```shell
+    ```bash title="Update: my_agent/.env"
     echo 'export GOOGLE_API_KEY="YOUR_API_KEY"' > .env
```

Comment:
```suggestion
    echo 'GOOGLE_API_KEY="YOUR_API_KEY"' > .env
```

These are read in as key-value pairs, so no need to export to the current environment.